### PR TITLE
New in doc: November 24

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | N/A |
| Versions      | 4.6, master |
| Edition       | all |

Update new in doc list.
Entry point for review:
https://ez-systems-developer-documentation--2531.com.readthedocs.build/en/2531/resources/new_in_doc/

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date

### CODE REVIEW

File: docs/resources/new_in_doc.md [hunk 1]
```diff
@@ -6,6 +6,79 @@ description: Overview of major recent additions to Ibexa DXP documentation.
...
+## Personalization
+
+- Added dynamic attribute submodels information by:
+    - mentioning them in [customizing the recommendation request](https://doc.ibexa.co/en/master/personalization/api_reference/recommendation_api/#customizing-the-recommendation-request) instructions
+    - describing them in [user documentation](https://doc.ibexa.co/projects/userguide/en/latest/personalization/recommendation_models/#dynamic-attributes)
+- Added time-slot based models information by:
+    - changing the list of parameters available when [customizing the recommendation request](https://doc.ibexa.co/en/master/personalization/api_reference/recommendation_api/#customizing-the-recommendation-request)
+    - describing them in [user documentation](https://doc.ibexa.co/projects/userguide/en/master/personalization/recommendation_models/#time-slot-based-models)
+
+- Updated configuration details (including endpoint addresses nad code examples) in multiple how-to articles:
```
Comment (mnocon) [inline docs/resources/new_in_doc.md:37]:
```suggestion
- Updated configuration details (including endpoint addresses and code examples) in multiple how-to articles:
```

File: docs/resources/new_in_doc.md [hunk 2]
```diff
@@ -6,6 +6,79 @@ description: Overview of major recent additions to Ibexa DXP documentation.
...
+    - [Tracking integration](https://doc.ibexa.co/en/master/personalization/tracking_integration/)
+    - [Track events with ibexa-tracker.js](https://doc.ibexa.co/en/master/personalization/tracking_with_ibexa-tracker/)
+
+### PIM
+
+- Updated the [Product API](https://doc.ibexa.co/en/master/pim/product_api/) article by fixing method signatures and adding links to the PHP API reference
+
+### PHP API
+
+- Added missing classes in the following namespaces:
```
Comment (mnocon) [inline docs/resources/new_in_doc.md:49]:
```suggestion
- The following new classes are now part of the public PHP API:
```
The classes were introduced with a bugfix release of Ibexa DXP (didn't exist before, exist now), so I'm not sure about calling them "missing"

**Review by mnocon — CHANGES_REQUESTED** [review]
Thank you!

A couple of small suggestions - request changes is only because changes from the other PR are included here, please revert https://github.com/ibexa/documentation-developer/pull/2531/commits/3522c323f4a179ca10ed330dc613e7cbfc968b19 (or remove in other way)

**Comment by dabrt** [comment]
@mnocon Thank you for the comments, I reverted the changes that got in inadvertently merged in from another branch, and it looks ok now.

**Review by juskora — APPROVED** [review]

**Pull request merged**